luigi/tasks/config.py

```diff
@@ -300,8 +300,51 @@ def ensembl_export(self, *args):
 
     def go(self, *args):
         return self.ftp('go', *args)
+      
+    def bed(self, *args):
+        return self.ftp('bed', *args)
 
+      
+class genome_mapping(luigi.Config):  # pylint: disable=C0103,R0904
+    base = PathParameter(default='/tmp')
 
+    def genomes(self, *args):
+        path = os.path.join(self.base, *args)
+        if not os.path.exists(path):
```

Comment:
I have an `task.utils.files.atomic_output` helper that will create a path if needed, might be better to use this. It is used like:

```python
with atomic_output(self.output) as raw:
    some_function(raw)
```

Reply:
I'll keep it in mind in the future. I think `atomic_output` won't work in this case because I need to create and return paths, not files